doc/how_to/branding.md

```diff
@@ -0,0 +1,385 @@
+# How to Brand Your Panel Material-UI Application
```

Comment:
I cannot figure out how to set the header background. It used to work but not any more. See picture above. Please note that the header color depends on the theme (light, dark).

The guide https://panel-material-ui.holoviz.org/reference/page/Page.html did not help me as I could not get those examples working here.

Can you help @philippjfr ?